lib/index.js

```diff
@@ -114,27 +115,21 @@ Nanigans.prototype.track = function(track) {
       params.products.value = data.value;
       params.products.sku = data.sku;
       params.products = qs.stringify(params.products);
-      this.load('purchase', params);
+      params.userId ? this.load('purchase', params) : this.load('purchase_no_user_id', params);
       break;
     case 'user':
       switch (event.name) {
-      case 'product':
-        params.sku = data.sku;
-        break;
       case 'add_to_cart':
-        params.products.qty = data.qty;
```

Comment:
Because for the old line 24, {{ products }} wouldn't populate with anything because an array of products wasn't passed, so the keys for params that I removed here were not used. Tong also confirmed that the new add_to_cart tags look good as [mentioned here](https://cl.ly/0u033c0d1Z17).